README.md

```diff
@@ -26,23 +26,51 @@ Most of Data.gov discussions happen at [Data.gov github](https://github.com/gsa/
 - [Docker](https://docs.docker.com/install/) v18+
 - [Docker Compose](https://docs.docker.com/compose/) v1.24+
 
+
 ### Setup
 
 Build the docker containers.
 
-    $ docker-compose build
-
-Run the docker containers.
-
-    $ docker-compose up
+    $ make build
 
-Activate all the installed plugins and theme.
+Configure WordPress by installing the composer dependencies, installing
+WordPress, and configuring plugins and theme.
 
-    $ docker-compose exec app wp core install --url=http://localhost:8000 \
-      --title=Data.gov --admin_user=admin --admin_email=admin@example.com --allow-root
-    $ docker-compose exec app wp plugin activate --all --allow-root
-    $ docker-compose exec app wp theme activate roots-nextdatagov --allow-root
+    $ make configure
 
 Open your browser to [localhost:8000](http://localhost:8000/).
```

Comment:
This won't work until you run `make up`; should move this lower and/or include that step here.

Reply:
The thinking was that we switch to `docker-compose run`, so there doesn't need to be an existing container, but the volumes would be populated so that when `make up` is run, it's ready to go.

However, I'm looking at the `.gitignore` and realizing there are some more volumes that need to be setup, otherwise we'll run into the same kind of bug that @ydave-reisys saw, so I will test with `git clean -xdf` before pushing a fix.

Reply:
I just rebased onto develop, which has the volumes updated...